homeassistant/components/switchbot_cloud/config_flow.py

```diff
@@ -7,16 +7,20 @@
 import voluptuous as vol
 
 from homeassistant.config_entries import ConfigFlow, ConfigFlowResult
-from homeassistant.const import CONF_API_KEY, CONF_API_TOKEN
+from homeassistant.const import CONF_API_KEY, CONF_API_TOKEN, CONF_SCAN_INTERVAL
+from homeassistant.helpers import config_validation as cv
 
-from .const import DOMAIN, ENTRY_TITLE
+from .const import DEFAULT_SCAN_INTERVAL_SECONDS, DOMAIN, ENTRY_TITLE
 
 _LOGGER = getLogger(__name__)
 
 STEP_USER_DATA_SCHEMA = vol.Schema(
     {
         vol.Required(CONF_API_TOKEN): str,
         vol.Required(CONF_API_KEY): str,
+        vol.Required(
```

Comment:
We don't allow exposing/modifying the scan interval. They should be set to sane default values.
If the user wants a short update interval, they can use the service [`homeassistant.update_entity`](https://www.home-assistant.io/integrations/homeassistant/#service-homeassistantupdate_entity)

Reply:
Fair enough. I've reverted this.